src/rewrite/vct/rewrite/exc/EncodeBreakReturn.scala

```diff
@@ -174,10 +174,10 @@ case class EncodeBreakReturn[Pre <: Generation]() extends Rewriter[Pre] {
 
               if(needReturn(method)) {
                 val returnField = new InstanceField[Post](dispatch(method.returnType), Nil)(ReturnField)
-                val returnClass = new Class[Post](Seq(returnField), Nil, tt)(ReturnClass)
+                val returnClass = new Class[Post](Nil, Seq(returnField), Nil, tt)(ReturnClass)
```

Comment:
Aww, surely you saw the opportunity here :D

Reply:
Oh! That's a good idea, also a good way to indirectly test the system much more thoroughly. I'll save that for later.